catalog/views.py

```diff
@@ -3,8 +3,26 @@
 from catalog.constants import NUMBER_PAGINATE_BY, LOAN_STATUS_M, LOAN_STATUS_A, LOAN_STATUS_O, LOAN_STATUS_R
 from django.views import generic
 from django.shortcuts import get_object_or_404
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.decorators import permission_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.decorators import user_passes_test
```

Comment:
import cùng package/module thì nên đặt cùng dòng e nha
```suggestion
from django.contrib.auth.decorators import login_required, permission_required, user_passes_test
from django.contrib.auth.mixins import LoginRequiredMixin
```